services/provider/proto/provider.proto

```diff
@@ -15,6 +15,9 @@ service OCSProvider {
   // GetStorageConfig RPC call to generate the json config for connecting to storage provider cluster
   rpc GetStorageConfig(StorageConfigRequest)
   returns (StorageConfigResponse){}
+  // GetStorageInfo RPC call to generate the json config for connecting to storage provider cluster
+  rpc GetStorageInfo(StorageInfoRequest)
```

Comment:
As this is a new RPC function, can we call it getClientDesiredState? This will also eliminate the need to keep on adding a desired prefix to members

Of course all other names need to align with this rename